README.md

```diff
@@ -145,6 +153,226 @@ readData(dataInput) {
 }
 ```
 
+# Setting Up Client Network
+
+All network related configuration of Hazelcast Node.js Client is performed via the `network` element in the declarative configuration file, or in the class ClientNetworkConfig when using programmatic configuration. Let’s first give the examples for these two approaches. Then we will look at its sub-elements and attributes.
+
+### Declarative Client Network Configuration
+
+Here is an example of configuring network for Node.js Client declaratively.
+
+```json
+{
+    "network": {
+        "clusterMembers": [
+            "10.1.1.21",
+            "10.1.1.22:5703"
+        ],
+        "smartRouting": true,
+        "redoOperation": true,
+        "connectionTimeout": 6000,
+        "connectionAttemptPeriod": 5000,
+        "connectionAttemptLimit": 5
+    }
+}
+```
+
+### Programmatic Client Network Configuration
+
+Here is an example of configuring network for Node.js Client programmatically.
+
+```javascript
+var clientConfig = new Config.ClientConfig();
+clientConfig.networkConfig.addresses.push('10.1.1.21', '10.1.1.22:5703');
+clientConfig.networkConfig.smartRouting = true;
+clientConfig.networkConfig.redoOperation = true;
+clientConfig.networkConfig.connectionTimeout = 6000;
+clientConfig.networkConfig.connectionAttemptPeriod = 5000;
+clientConfig.networkConfig.connectionAttemptLimit = 5;
+```
+
+## 1. Providing the Member Addresses
+
+Address list is the initial list of cluster addresses to which the client will connect. The client uses this
+list to find an alive member. Although it may be enough to give only one address of a member in the cluster
+(since all members communicate with each other), it is recommended that you give the addresses for all the members.
+
+**Declarative:**
+
+```json
+{
+    "network": {
+        "clusterMembers": [
+            "10.1.1.21",
+            "10.1.1.22:5703"
+        ]
+    }
+}
+```
+
+**Programmatic:**
+
+```javascript
+var clientConfig = new Config.ClientConfig();
+clientConfig.networkConfig.addresses.push('10.1.1.21', '10.1.1.22:5703');
+```
+
+If the port part is omitted, then 5701, 5702 and 5703 will be tried in random order.
+
+You can provide multiple addresses with ports provided or not, as seen above. The provided list is shuffled and tried in random order. Its default value is `localhost`.
```

Comment:
You can specify multiple addresses with or without port information as seen above. The...